### DISCUSSION TITLE
test app

### DISCUSSION THREAD

**Discussion by alephbang** [outline/outline#5500] [Show and tell]
App test have problem. I found https://stackoverflow.com/questions/32911630/how-do-i-deal-with-localstorage-in-jest-tests
Jest doesn't need to initialize, set mockup localstorage. After I comment out, test passed.
> 
 FAIL   app  app/components/PaginatedList.test.tsx
  ● Test suite failed to run

    TypeError: _localStorage.default is not a constructor

      12 | });
      13 |
    > 14 | global.localStorage = new localStorage;
         |                       ^
      15 |
      16 | require("jest-fetch-mock").enableMocks();
      17 |

      at Object.<anonymous> (app/test/setup.ts:14:23)

**Comment by tommoor** [comment]
It's fine here and on CI 🤷 

<img width="728" alt="image" src="https://github.com/outline/outline/assets/380914/60708511-43a4-4180-a3c9-ced6b5f09280">
